server/server.go

```diff
@@ -92,6 +93,8 @@ func New(cfg Config) (*Server, error) {
 		return nil, err
 	}
 
+	var loggingInterceptor func(context.Context, interface{}, *grpc.UnaryServerInfo, grpc.UnaryHandler) (interface{}, error)
```

Comment:
This would be simpler as:
```
loggingInterceptor := middleware.ServerLoggingInterceptor
if cfg.TruncatedServerLogging {
	loggingInterceptor = middleware.TruncatedServerLoggingInterceptor
}
```